cinnabar/measurements.py

```diff
@@ -52,19 +54,52 @@ class Measurement(DefaultModel):
 
     @classmethod
     def from_experiment(cls,
-                        pIC50: float,
+                        IC50: float,
```

Comment:
looking at the uncertainty about units below, maybe this, uncertainty and temperature should be given in `unit.Quantity` values.  There's some cool stuff you can do:

```python
In [1]: from openff.units import unit

In [2]: 5.0 * unit.micromolar
Out[2]: 5.0 <Unit('micromolar')>

In [3]: m = 5.0 * unit.micromolar

In [4]: m.is_compatible_with(unit.molar)
Out[4]: True

In [5]: m.is_compatible_with(unit.kelvin)
Out[5]: False

In [6]: m.to(unit.nanmolar)
Out[6]: 4999.999999999999 <Unit('nanomolar')>

```